hooks/templatedatavalidator.js

```diff
@@ -25,13 +26,50 @@ module.exports = function (pageData, partials) {
  * Validates global config for the page template
  * 
  * @param {Object} globalConfig 
+ * @param {Object} JAMBO_INJECTED_DATA
  * @returns {boolean}
  */
-function isGlobalConfigValid(globalConfig) {
-  if (!globalConfig.experienceKey) {
+function isGlobalConfigValid(globalConfig, JAMBO_INJECTED_DATA) {
+  const { experienceKey } = globalConfig;
+  if (!experienceKey) {
     error('Missing Info: no experienceKey found.');
     return false;
   }
+  if (globalConfig.useJWT || globalConfig.apiKey) {
+    return true;
+  }
+
+  const injectedDataForExperience = JAMBO_INJECTED_DATA.answers.experiences[experienceKey];
```

Comment:
Could we use the `?` syntax here? It's possible something on Spruce's end could cause there to be no `experiences` under `answers`. If that were the case, the validator would throw an NPE.

Reply:
I think blowing up is probably fine for now. The NPE will be easy to diagnose I expect.